### DISCUSSION TITLE
Docker image up

### DISCUSSION THREAD

**Discussion by appleimperio** [awawa-dev/HyperHDR#126] [Ideas]
First of all thank you Awawa again for a great job. This version looks amazing, if someone is interested on a Docker image of the latest version (v18) here is the link https://hub.docker.com/r/nocturno/hyper-hdr . This is an AARCH64 docker image. Any problem related to the Docker image or if you prefer to build your own image here is the place https://github.com/appleimperio/docker-hyperhdr.

**Comment by awawa-dev** [comment]
Thanks @appleimperio  👍 
I move it to the Discussion

**Comment by awawa-dev** [comment]
Added to v18 release info

**Comment by appleimperio** [comment]
docker ARM v7 added

**Comment by nazar-pc** [comment]
@appleimperio, what about upstreaming at least the Dockerfile? Although it is easy to build and automatically publish images to GitHub Container Registry from CI.

**Reply by nazar-pc** [reply]
Oh, I just noticed https://github.com/awawa-dev/HyperDockerBuilder exists. Not that obvious initially, but at least it is something.

UPD: It is only for building, not running though.
